src/cloud-api-adaptor/podvm/Makefile

```diff
@@ -55,10 +56,14 @@ ifeq ($(ARCH),s390x)
 	$(eval OPTS += -var machine_type=${QEMU_MACHINE_TYPE_${ARCH}})
 	$(eval OPTS += -var cpu_type=max)
 	$(eval OPTS += -var os_arch=s390x)
+endif
+ifeq ($(ARCH),aarch64)
+	$(eval OPTS += -var machine_type=${QEMU_MACHINE_TYPE_${ARCH}})
```

Comment:
Shouldn't we also set `os_arch`?

Reply:
I didn't encounter any indication I needed to set that